packages/next/build/swc/src/styled_jsx/utils.rs

```diff
@@ -82,43 +82,49 @@ pub fn compute_class_names(
         span: DUMMY_SP,
         computed: false,
       }))),
-      args: dynamic_styles
-        .iter()
-        .map(|style_info| {
-          let hash_input = match &static_class_name {
-            Some(class_name) => format!("{}{}", style_info.hash, class_name),
-            None => style_info.hash.clone(),
-          };
-          ExprOrSpread {
-            expr: Box::new(Expr::Array(ArrayLit {
-              elems: vec![
-                Some(ExprOrSpread {
-                  expr: Box::new(string_literal_expr(&hash_string(&hash_input))),
-                  spread: None,
-                }),
-                Some(ExprOrSpread {
-                  expr: Box::new(Expr::Array(ArrayLit {
-                    elems: style_info
-                      .expressions
-                      .iter()
-                      .map(|expression| {
-                        Some(ExprOrSpread {
-                          expr: expression.clone(),
-                          spread: None,
-                        })
-                      })
-                      .collect(),
-                    span: DUMMY_SP,
-                  })),
-                  spread: None,
-                }),
-              ],
-              span: DUMMY_SP,
-            })),
-            spread: None,
-          }
-        })
-        .collect(),
+      args: vec![ExprOrSpread {
+        expr: Box::new(Expr::Array(ArrayLit {
+          elems: dynamic_styles
+            .iter()
```

Comment:
I think this can be changed to `.into_iter()` to avoid cloning.

Reply:
This doesn't seem to be working for me, I can change it to `into_iter()` but it doesn't allow me to remove any of the clones within. The hashes are used elsewhere later, so it seems like it might be necessary?

Reply:
Ah, my bad. Sorry.